csharp/extractor/Semmle.Extraction.CSharp/Entities/Expressions/Invocation.cs

```diff
@@ -84,6 +84,13 @@ protected override void PopulateExpression(TextWriter trapFile)
                 else
                     Context.ModelError(Syntax, "Unable to get name for dynamic call.");
             }
+            else
+            {
+                if (memberName is not null)
+                {
+                    trapFile.invocation_member_name(this, memberName);
```

Comment:
Why are all `dynamic` calls excluded?
A far as I understand, a call can be considered dynamic, if just one of the arguments are.

Reply:
That's a good point. We could include `dynamic_member_name` in `MethodCall::getACandidateTarget`, where we use `invocation_member_name`. But there's no need to store dynamic member names duplicately in the DB.

Reply:
I've done this in https://github.com/github/codeql/pull/14891/commits/82c675a29eecc5b01cd7f68494b10aa4ba34d2da.